### PR TITLE
minor doc update for Chunk Data Pack

### PR DESCRIPTION
**Pull request by AlexHentschel**
extending the documentation to be clear that State Proofs need to include all touched registers (read and written)

### CODE REVIEW

**Review by ramtinms — APPROVED** [review]

**Review by janezpodhostnik — APPROVED** [review]

File: model/flow/chunk.go [hunk 1]
```diff
@@ -60,14 +60,18 @@ func (ch *Chunk) Checksum() Identifier {
 
 // ChunkDataPack holds all register touches (any read, or write).
 //
-// Note that we have to capture a read proof for each write before updating the registers.
-// `Proof` includes proofs for all registers read to execute the chunk.
+// Note that we have to include a merkle path for all registers touched (read or written) for
```
Comment (zhangchiqing) [inline model/flow/chunk.go:63]:
```suggestion
// Note that we have to include merkle paths as storage proof for all registers touched (read or written) for
```

File: model/flow/chunk.go [hunk 2]
```diff
@@ -60,14 +60,18 @@ func (ch *Chunk) Checksum() Identifier {
...
+// the _starting_ state of the chunk (i.e. before the chunk computation updates the registers).
```
Comment (zhangchiqing) [inline model/flow/chunk.go:64]:
```suggestion
// the _starting_ state of the chunk (i.e. before the chunk computation updates the registers).
// For instance, if an execution state contains three registers: { A: 1, B: 2, C: 3}, and a certain 
// chunk has a tx to execute A = A + B, then its chunk data pack should include the merkle 
// paths for { A: 1, B: 2 } as storage proof.
```

**Review by zhangchiqing — APPROVED** [review]

File: model/flow/chunk.go (hunk 2 above)
Comment (bluesign) [reply model/flow/chunk.go:64]: in `A = B` case is it just `{B: 2}` as storage proof ?

Comment (zhangchiqing) [reply model/flow/chunk.go:64]: No, in `A=B` case, even though A is not read, it still gets written. Note, the storage proof includes both reads and writes, that's why we call it "touched".

**Comment by AlexHentschel** [comment]
all suggestions applied

**Pull request merged**